src/ChainSafe.Gaming.Unity/link.xml

```diff
@@ -19,10 +19,7 @@
     <assembly fullname="ChainSafe.Gaming.InProcessSigner"/>
     <assembly fullname="ChainSafe.Gaming.InProcessTransactionExecutor"/>
     <assembly fullname="ChainSafe.Gaming.NetCore"/>
-    <assembly fullname="ChainSafe.Gaming.WalletConnect">
```

Comment:
Should we Preserve the entire assembly? We only need it for the WC JsonRPC methods, or is this causing problems?

Reply:
Yeah that was causing the problem unfortunately, because some parts of the WalletConnect.Model were also being stripped. So I wanted to be better safe than sorry and preserved everything.